source/ui/Components/Register/DefaultRegister.qml

```diff
@@ -95,4 +113,74 @@ TextField {
         return registerContentString;
     }
 
+
+    style: TextFieldStyle{
+        background: Rectangle {
+            id: rect
+            x: registerTextField.x
+            y: registerTextField.y
+            width: registerTextField.width
+            height: registerTextField.height
+            color: registerTextField.backgroundColor
+            radius: 2
+            border.color: "lightgray"
+            border.width: 1
+        }
+    }
+
+    //can be used for blinking
+    /*Component{
```

Comment:
As this component is not used, I would delete this before merging to master, as this code can not be properly tested by reviewers at the moment. Futhermore, the structure of this seems strange, because it is nested dynamic loading (a component (which has to be created dynamically) loads another component...that should be possible without this indirection). As the highlights are changed quite often, this much dynamic loading might be very inefficient.